File: Contests/Round61/B_Discounts.java
import java.util.Arrays;
import java.util.Scanner;

//CF считает что java медленная... на Python все работает

public class B_Discounts {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        int[] a = new int[n];
        long sum = 0;
        for (int i = 0; i < n; i++) {
            a[i] = scanner.nextInt();
            sum += a[i];
        }
        Arrays.sort(a);
        int m = scanner.nextInt();
        for (int i = 0; i < m; i++) {
            int q = scanner.nextInt();
            System.out.println(sum - a[n - q]);
        }
    }
}

File: Contests/Round61/B_Discounts.py
n = int(input())
a = input().split(' ')

sum = 0
for i in range(n):
    t = int(a[i])
    sum += t
    a[i] = t

a.sort()

m = int(input())
q = input().split(' ')
for i in range(m):
    print(sum - a[n - int(q[i])])
